websocket/src/websocket.cpp

```diff
@@ -239,21 +242,26 @@ static void CloseConnection(WebsocketConnection* conn)
     // we want it to send this message in the polling
     if (conn->m_State == STATE_CONNECTED) {
 #if defined(HAVE_WSLAY)
-        WSL_Close(conn->m_Ctx);
+    WSL_Close(conn->m_Ctx);
+#else
+    emscripten_websocket_close(conn->m_WS, 1000, "CloseConnection");
 #endif
     }
 
     SetState(conn, STATE_DISCONNECTED);
 }
 
-static int FindConnection(WebsocketConnection* conn)
+bool IsConnectionValid(WebsocketConnection* conn)
```

Comment:
The only way this function is used is to check if the connection is valid (ie hasn't been destroyed).

Reply:
It can go back to being `static`